android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java

```diff
@@ -4285,6 +4277,52 @@ public static boolean handleOpenDocError(ViewerComponent component) {
         return false;
     }
 
+    public static void handleOnConfigurationChanged(ViewerComponent component) {
+        if (component == null) {
+            return;
+        }
+        if (component.getAppNavRightBarItems() != null) {
+            PdfViewCtrlTabHostFragment2 hostFragment = component.getPdfViewCtrlTabHostFragment();
```

Comment:
Majority of this code is duplicated within `handleDocumentLoaded`

Could you please look to merge this into one function and call in two places?

Reply:
fixed